libs/shared/routes/src/lib/sub-router/clusters.router.ts

```diff
@@ -1,11 +1,15 @@
 export const CLUSTERS_URL = (organizationId = ':organizationId') => `/organization/${organizationId}/clusters`
 export const CLUSTERS_GENERAL_URL = '/general'
+export const CLUSTERS_CREATION_URL = '/create'
 
 export const CLUSTER_URL = (organizationId = ':organizationId', clusterId = ':clusterId') =>
   `/organization/${organizationId}/cluster/${clusterId}`
 
-export const CLUSTER_SETTINGS_URL = '/settings'
+// subrouter for cluster create steps /create/general /create/settings etc...
+export const CLUSTERS_CREATION_GENERAL_URL = '/general'
 
+// settings
+export const CLUSTER_SETTINGS_URL = '/settings'
```

Comment:
You don't want to put these routes in a separate files since they are not concerning the same lib ? It's the same domain (cluster) so it makes also sense to have them here. As you prefer!